### ISSUE TITLE
logError() is broke

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7848]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)| @wsanchez reported|
|-|-|
|Trac ID|trac#7848|
|Type|enhancement|
|Created|2015-04-14 16:37:13Z|
|Branch|https://github.com/twisted/twisted/tree/log-defer-7848|

`defer.logError()` calls `log.err()` which doesn't exist; it should be `log.error()`.

<details><summary>Searchable metadata</summary>

```
trac-id__7848 7848
type__enhancement enhancement
reporter__wsanchez wsanchez
priority__normal normal
milestone__New_Logging_System New Logging System
branch__branches_log_defer_7848 branches/log-defer-7848
branch_author__wsanchez wsanchez
status__closed closed
resolution__fixed fixed
component__logger logger
keywords__ 
time__1429029433238494 1429029433238494
changetime__1481165855745880 1481165855745880
version__None None
owner__wsanchez wsanchez

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez set owner to @wsanchez<br>@wsanchez set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez removed owner<br>@wsanchez set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_3"></a>|@ralphm set owner to @wsanchez|
|-|-|

 * You can instantiate an exception like so: `failure.Failure(RuntimeError())`
 * Please use flushLoggedErrors in the second test.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_4"></a>|@wsanchez set status to `closed`|
|-|-|

(In [44420]) Merge branches/log-defer-7848: logDefer() raises due to undefined attribute.

Author: wsanchez
Reviewer: ralphm
Fixes: [#7848](https://github.com/twisted/twisted/issues/7848)
